server.js

```diff
@@ -9,13 +9,23 @@ const CLIENT_SDK_ID = 'e2d30c07-90e8-4e68-bdd1-691b4242d0f9';
 const server = new Hapi.Server();
 const yoti_key = fs.readFileSync(path.join(__dirname, './keys/yoti-key.pem'));
 
-var client = new Yoti(CLIENT_SDK_ID, yoti_key);
+const client = new Yoti(CLIENT_SDK_ID, yoti_key);
 
 const tls = {
   key: fs.readFileSync(path.join(__dirname, './keys/key.pem'), 'utf8'),
   cert: fs.readFileSync(path.join(__dirname, './keys/cert.pem'), 'utf8'),
 };
 
+let users = {};
+
+function userSignIn (user) {
```

Comment:
Any reason why this function is being declared globally in server.js, outside the handler?